docs/config/index.md

```diff
@@ -33,7 +33,7 @@ Include globs for test files
 ### exclude
 
 - **Type:** `string[]`
-- **Default:** `['node_modules', 'dist', '.idea', '.git', '.cache']`
+- **Default:** `['**/node_modules/**', '**/dist/**', '**/cypress/**', '**/.{idea,git,cache,output,temp}/**']`
```

Comment:
I don't think there's anything wrong with the previous default and you can always add extend to add yours in your config.

Reply:
I only changed the documentation. The implementation has already been updated.

Reply:
I was trying to add `**/cypress/**` to my project exclude, but later found out that it was already excluded by default. So I realized the documentation on the website is out of date.